src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -225,6 +212,59 @@ protected function createConfigurationDefinition()
                 ->setAllowedValues([$constantChecker])
                 ->setDefault(['null', 'false', 'true'])
                 ->getOption(),
+            (new FixerOptionBuilder('scope', 'Only fix constant invocations that are made within a namespace or fix all.'))
```

Comment:
As we've added this option, can we also add some a test or two to check that `'scope' => 'all'` works correctly

Reply:
well, all the non-modified tests are running with the `all` scope